### ISSUE TITLE
Revisit $fish_user_paths usage

### ISSUE THREAD

**Issue by jorgebucaran** [jorgebucaran/nvm.fish#114]
If nvm.fish tries to `set -U fish_user_paths ... $fish_user_paths`, the following error can occur if `$fish_user_paths` has global scope:

```
set: Universal variable 'fish_user_paths' is shadowed by the global variable of the same name.
```

See: https://github.com/jorgebucaran/nvm.fish/pull/107

**Comment by nathan-gilbert** [comment]
I am getting this error

**Comment by jorgebucaran** [comment]
Just set the variable to universal scope for now.

**Comment by jackwestmoretab** [comment]
I think that a simple fix for this might be to change the offending line to: `set fish_user_paths "$nvm_config/$ver/bin" $fish_user_paths`

i.e., remove the `-U` setting. This simply appends `"$nvm_config/$ver/bin"` to whatever is in the user's `fish_user_paths` already. Works on my system.

**Comment by jorgebucaran** [comment]
@jackwestmoretab Yes, makes sense. 👍

**Comment by jackwestmoretab** [comment]
@jorgebucaran would you like me to make a Pull Request to this effect?

**Comment by jorgebucaran** [comment]
@jackwestmoretab Yes, go ahead, please! 💯

**Comment by jackwestmoretab** [comment]
@jorgebucaran https://github.com/jorgebucaran/nvm.fish/pull/121

**Comment by arbourd** [comment]
Anyway to use this great implementation of nvm in the meantime? I only get errors even if I set it on my path manually.

My work around was change the script itself in `functions`.

**Comment by jorgebucaran** [comment]
@arbourd PR https://github.com/jorgebucaran/nvm.fish/pull/123.

**Comment by jorgebucaran** [comment]
I just released [2.0](https://github.com/jorgebucaran/nvm.fish/releases/tag/2.0.0). We're no longer using `fish_user_paths`. Installing or activating a version now changes the current shell only, for which we modify the PATH directly.

@arbourd @nathan-gilbert @jackwestmoretab @sh78 👋
